design/FY2023/NFP-evapCoolerRHcontrol.md

```diff
@@ -0,0 +1,274 @@
+        \key EvaporativeCooler:Direct:ResearchSpecial
+        \key EvaporativeCooler:Indirect:CelDekPad
+        \key EvaporativeCooler:Indirect:WetCoil
+        \key EvaporativeCooler:Indirect:ResearchSpecial
+    A12, \field First Evaporative Cooler Object Name
+        \required-field
+        \type object-list
+        \object-list EvapCoolerNames
+    A13, \field Second Evaporative Cooler Object Type
+        \note optional, used for direct/indirect configurations
+        \note second cooler must be immediately downstream of first cooler, if present
+        \type choice
+        \key EvaporativeCooler:Direct:CelDekPad
+        \key EvaporativeCooler:Direct:ResearchSpecial
+        \key EvaporativeCooler:Indirect:CelDekPad
+        \key EvaporativeCooler:Indirect:WetCoil
+        \key EvaporativeCooler:Indirect:ResearchSpecial
+    A14, \field Second Evaporative Cooler Name
+        \note optional, used for direct/indirect configurations
+        \type object-list
+        \object-list EvapCoolerNames
+    A15, \field Design Specification ZoneHVAC Sizing Object Name
+        \note Enter the name of a DesignSpecificationZoneHVACSizing object.
+        \type object-list
+        \object-list DesignSpecificationZoneHVACSizingName
+    N4, \field Shut Off Relative Humidity
+        \note Zone relative humidity above which the evap cooler is shut off.
+        \type real
+        \minimum 0.00
+        \maximum 100.00
+        \units percent
+    N5; \field Turn On Relative Humidity
+        \note Zone relative humidity below which the evap cooler is allowed to turn on.
+        \type real
+        \minimum 0.00
+        \maximum 100.00
+        \units percent
+
+For non-zone-equipment direct evaporative coolers, a generic *AvailabilityManager:\** object will be added, both to implement the relative humidity-dependent availability, and to facilitate future expansion of availability manager functionalities.
+
+    AvailaibilityManager:SystemNodeConditions,
+        A1 , \field Name
+        \required-field
+        \type alpha
+        \reference SystemAvailabilityManagers
+    A2,  \field Action Type
+        \required-field
+        \type choice
+        \key TurnOn
+        \key TurnOff
+    A3 , \field Sensor Node Name
+        \note The system node where the conditions are monitored.
+        \required-field
+        \type node
+    A4 , \field Limit 1 Condition
+        \required-field
+        \type choice
+        \key Temperature
+        \key RelativeHumidity
+        \key HumidityRatio
+        \key DewPoint
+        \key Enthalpy
+        \key Schedule
+    A5,  \field Limit 1 Type
+        \required-field
+        \type choice
+        \key Maximum
+        \key Minimum
+        \note When "Maximum" is chosen, then the action is taken when the value
+        \note of A4 is less than the threshold. When "Minimum" is chosen, then the
+        \note action is taken when the value of A4 is greater than the threshold.
```

Comment:
These notes should be revised. When "Maximum" is chosen, then the action is taken when the value of A4 is greater than the threshold. When "Minimum" ... less than ... Also, doesn't it depend on which type of equipment uses this avail manager and then some combinations would not make sense. What if a dehumidifier using RelativeHumidity tried to use TurnOff and Maximum? or TurnOn and Minimum? Or if the EvapCooler got the inputs backwards, as RelativeHumidity, TurnOff and Minimum. Would these backward inputs be ignored or just do what the user specifies? I guess doing what the user specified would make the most sense since the avail manager does not know what type of equipment is being controlled.

Reply:
@rraustad per @mjwitte 's suggestion, we decided to leave out the generic availability manager and only modify the ZoneHVAC object in this PR.